suricata.yaml.in

```diff
@@ -1179,6 +1179,10 @@ legacy:
 #   - reject
 #   - alert
 
+# Define maximun number of possible alerts that can be triggered for the same
```

Comment:
nit: `maximun` :stuck_out_tongue:

Reply:
ugh I can't believe this is here again. x_x D: